mmv1/products/looker/Instance.yaml

```diff
@@ -0,0 +1,362 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'Instance'
```

Comment:
nit: Do you mind adding reference links similar to https://github.com/GoogleCloudPlatform/magic-modules/blob/887b7798751d630e771d3b0eff751c2c892c2365/mmv1/products/compute/Address.yaml#L35-L39?

Reply:
Added